src/markdown-pages/build-apps/add-visualizations-to-dashboard.mdx

```diff
@@ -0,0 +1,144 @@
+---
+path: '/build-apps/add-visualizations-to-dashboard'
+duration: '10 min'
+title: 'Add custom visualizations to your dashboards'
+template: 'GuideTemplate'
+description: 'Add your custom visualization to a New Relic dashboard'
+resources:
+  - title: Add custom visualizations to your dashboards
+    url: '/build-apps/add-visualizations-to-dashboard'
+  - title: New Relic One VSCode extension
+    url: https://marketplace.visualstudio.com/items?itemName=new-relic.nr1
+  - title: 'Build on New Relic One'
+    url: https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application
+  - title: New Relic VSCode extension pack
+    url: https://marketplace.visualstudio.com/items?itemName=new-relic.new-relic-extension-pack
+tags:
+  - nr1 cli
+  - NR One Catalog
+  - visualizations
+---
+
+<Intro>
+
+This guide builds off of the previous [Customize visualizations with configuration](build-apps/customize-visualizations-with-configuration) guide. If you haven't followed that guide, please start there as this guide assumes you have the code you built there to get started.
```

Comment:
What are you seeing when you click the link? 404 page? I've just tested it a few times and it's working for me.

Reply:
Yeah, a 404. It looks like it's because there's no leading slash, because the resulting url is:

```
/build-apps/add-visualizations-to-dashboard/build-apps/customize-visualizations-with-configuration
```